tutorials/llm/5_llm_slots.py

```diff
@@ -0,0 +1,121 @@
+# %% [markdown]
+"""
+# LLM: 5. LLM Slots
+
+If we want to retrieve some information from user input like name, address or
+email we can simply use Chatsky's Slot system and user regexes or other formally
+specified data retrieval techniques. But if the data is more finicky to get or
+not explicitly presented in utterance we
+encourage you to utilize Chatsky LLM Slots.
+In this tutorial we will see how we can set up Slots that uses LLM's under
+the hood to extract more obscure information from users input.
+"""
+# %pip install chatsky[llm] langchain-openai
+# %%
+from chatsky import (
+    RESPONSE,
+    TRANSITIONS,
+    PRE_TRANSITION,
+    GLOBAL,
+    LOCAL,
+    Pipeline,
+    Transition as Tr,
+    conditions as cnd,
+    processing as proc,
+    responses as rsp,
+)
+from langchain_openai import ChatOpenAI
+
+from chatsky.utils.testing import (
+    is_interactive_mode,
+)
+from chatsky.slots.llm import LLMSlot, LLMGroupSlot
+from chatsky.llm import LLM_API
+
+import os
+
+openai_api_key = os.getenv("OPENAI_API_KEY")
+
+# %% [markdown]
+"""
+In this example we define LLM Group Slot with two LLM Slots in it.
+Both of them can be used separately just as regular slots,
+but if you are going to extract several LLM Slots simultaneously
+we encourage you to put them in LLM Group Slot for optimization and convenience.
+
+In the `LLMSlot.caption` parameter you should put description of a data piece
+you want to retrieve. More specific descriptions will yield better results,
+especially when using smaller models.
+
+Note that we are passing the name of the model
+ from pipeline.models dictionary to LLMGroupSlot.model field.
```

Comment:
I didn't get it! `pipeline.models` and `LLMGrouptSlot.model` seem not to be mentioned elsewhere